app/src/main/java/com/ciandt/dragonfly/example/models/Project.kt

```diff
@@ -14,6 +14,8 @@ data class Project(
         var versions: List<Version> = emptyList()
 ) : KParcelable {
 
+    fun hasVersion(): Boolean = versions.isNotEmpty()
```

Comment:
I think hasAnyVersion() is better here. In case we want to add an hasVersion(version) later.